app/src/main/java/org/schabi/newpipe/player/PopupVideoPlayer.java

```diff
@@ -663,6 +664,9 @@ public void onPlaying() {
             lockManager.acquireWifiAndCpu();
 
             hideControls(DEFAULT_CONTROLS_DURATION, DEFAULT_CONTROLS_HIDE_TIME);
+
+            // Check for new version
+            new CheckForNewAppVersionTask().execute();
```

Comment:
Why do you run the check here, too?

Reply:
From what I remember the check was not being performed in the pop-up mode. Hence, I had to put it here. I'll check once again and make changes if needed.